plugin/pkg/shoot/managedseed/admission.go

```diff
@@ -122,12 +124,47 @@ func (v *ManagedSeed) Validate(ctx context.Context, a admission.Attributes, _ ad
 		return nil
 	}
 
-	switch {
-	case a.GetName() == "":
-		return v.validateDeleteCollection(ctx, a)
-	default:
-		return v.validateDelete(ctx, a)
+	if a.GetOperation() == admission.Update {
+		return v.validateUpdate(ctx, a)
+	} else if a.GetOperation() == admission.Delete {
```

Comment:
You don't need the else here right? I guess it would be a bit cleaner without it.

Reply:
Makes sense, we can leave it as it is.